.travis.yml

```diff
@@ -1,16 +1,10 @@
-language: generic
-env:
-  global:
-    - export PATH="/opt/rakudo-pkg/bin:/opt/rakudo-pkg/share/perl6/site/bin:$PATH"
-addons:
-  apt:
-    sources:
-      - sourceline: 'deb https://dl.bintray.com/nxadm/rakudo-pkg-debs $(lsb_release -cs) main'
-        key_url: 'http://keyserver.ubuntu.com/pks/lookup?search=0x379CE192D401AB61&op=get'
-    packages:
-      - rakudo-pkg
+language: perl6
+perl6:
+  - latest
 install:
+  - rakudobrew build-zef
   - zef install Test::META
   - zef install --deps-only --/test .
```

Comment:
You may find that this doesn't install any ```test-depends``` if there are any.  Just something that might catch you out.